http-client/src/test/groovy/io/micronaut/http/client/JsonStreamSpec.groovy

```diff
@@ -209,7 +209,6 @@ class JsonStreamSpec  extends Specification {
         Publisher<Book> list();
 
         @Post(uri = "/count", processes = MediaType.APPLICATION_JSON_STREAM)
-        @SingleResult
         Publisher<LibraryStats> count(@Body Flux<Book> theBooks)
```

Comment:
breaking change, APPLICATION_JSON_STREAM doesn't make sense for parsing to single result. it gives an error now.